streaming/index.js

```diff
@@ -1338,11 +1347,12 @@ const startServer = async () => {
     channelNameToIds(request, channelName, params).then(({ channelIds }) => {
       removeSubscription(session, channelIds);
     }).catch(err => {
-      logger.error({err}, 'Unsubscribe error');
+      logger.error({err}, 'Websocket unsubscribe error');
 
       // If we have a socket that is alive and open still, send the error back to the client:
       if (websocket.isAlive && websocket.readyState === websocket.OPEN) {
-        websocket.send(JSON.stringify({ error: "Error unsubscribing from channel" }));
+        // TODO: Use a better error response here
+        websocket.send(JSON.stringify({error: "Error unsubscribing from channel" }));
```

Comment:
I haven't changed this to match `subscribeWebsocketToChannel` because something may be depending on this hardcoded error string here.